src/Refitter.Tests/PathParametersWithUrlTests.cs

```diff
@@ -0,0 +1,84 @@
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Refitter.Core;
+using Refitter.Tests.Build;
+using Xunit;
+
+namespace Refitter.Tests;
+
+public class PathParametersWithUrlTests
+{
+    private const string OpenApiSpec = @"
```

Comment:
This const doesn't seem to be used, but I can just remove that myself once this gets merged in